gradlew

```diff
@@ -214,11 +215,11 @@ fi
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
 DEFAULT_JVM_OPTS='"-Xmx64m" "-Xms64m"'
 
-# Collect all arguments for the java command;
-#   * $DEFAULT_JVM_OPTS, $JAVA_OPTS, and $GRADLE_OPTS can contain fragments of
-#     shell script including quotes and variable substitutions, so put them in
-#     double quotes to make sure that they get re-expanded; and
-#   * put everything else in single quotes, so that it's not re-expanded.
+# Collect all arguments for the java command:
+#   * DEFAULT_JVM_OPTS, JAVA_OPTS, JAVA_OPTS, and optsEnvironmentVar are not allowed to contain shell fragments,
```

Comment:
JAVA_OPTS is repeated. 

Why did we remove GRADLE_OPTS? It a legitimate environment variable: https://docs.gradle.org/current/userguide/build_environment.html

Reply:
@divijvaidya  thanks so much for this catch, I used to think that is is a missing from the Gradle wrapper update script I already restored this value in the last commit